src/wp-includes/interactivity-api/interactivity-api.php

```diff
@@ -66,7 +66,11 @@ function wp_interactivity_process_directives_of_interactive_blocks( array $parse
 
 	return $parsed_block;
 }
-add_filter( 'render_block_data', 'wp_interactivity_process_directives_of_interactive_blocks' );
+/**
+ * Uses a priority of 20 to ensure that other filters can add additional attributes to
+ * $parsed_block before the processing starts.
+ */
+add_filter( 'render_block_data', 'wp_interactivity_process_directives_of_interactive_blocks', 20, 1 );
```

Comment:
```suggestion

/*
 * Uses a priority of 20 to ensure that other filters can add additional attributes to
 * $parsed_block before the processing starts.
 */
add_filter( 'render_block_data', 'wp_interactivity_process_directives_of_interactive_blocks', 20, 1 );
```

Not sure the comment makes a lot of sense though. One could just use a priority < 10 otherwise, no? Why does 20 make things different?

Reply:
Usually, in Core,  that 10 is used. That's the reason we need one superior, we added 20, so anyone can make filters that have more priority that those ones but less than this one.

https://github.com/WordPress/gutenberg/blob/d1bbd9f9badb5a24866621ec9b4b0afb83bbb068/lib/block-supports/layout.php#L908

Ideally, this filter should be last one always, so we compare the final block version, as the comparison is done in the `render_block` filter, which is always after all the `render_block_data`, but  I guess putting a 100 may be too extreme.

Reply:
Just saw this comment thread, I voiced a similar concern in https://github.com/WordPress/wordpress-develop/pull/6245#discussion_r1519907466.

I don't think a 100 is too extreme. While that's still not a truly reliable solution, what we need here is to minimize the chance another filter callback can modify the data after this one. I'd recommend `PHP_INT_MAX`. But whatever you go with 100, 1000, 9999, etc., the higher the better. It's just a very weird problem.